Codeinwp/neve
# Feat/wp troubleshoot

### Summary
Added an Accordion for Site Health
Displays basic debug info for Neve:
1. Api Connectivity to api.themeisle.com as Yes ( when the site is reachable ) or NO along with the error info when is not.
2. Child theme files ( if using Child Theme ) as the list of the child theme files that they have in the child theme directory.
3. Custom CSS from Customizer, if any.


### Will affect visual aspect of the product
NO

![image](https://user-images.githubusercontent.com/23024731/124759434-4bee4080-df38-11eb-993e-f989de6a618c.png)

![image](https://user-images.githubusercontent.com/23024731/124759519-61636a80-df38-11eb-9835-08cedc797d34.png)

### Test instructions
1. Go to `/wp-admin/site-health.php` or *Tools* > *Site Health*
2. Check output 
3. Add `define( 'WP_HTTP_BLOCK_EXTERNAL', true );` to wp-config.php also remove transients using wp-cli command: `wp transient delete --all` check that the API access is blocked and it is being displayed
![image](https://user-images.githubusercontent.com/23024731/124760297-4513fd80-df39-11eb-8f39-5a602abe0d48.png)
4. Create a child theme and activate it.
5. Check that the Child theme files are being displayed
6. Add custom CSS in Customizer
7. Check that is being shown inside the debug tab
![image](https://user-images.githubusercontent.com/23024731/124761464-7fca6580-df3a-11eb-9415-b3d28edd153b.png)


Closes #2938.
